docs/hub/spaces-sdks-streamlit.md

```diff
@@ -76,31 +76,27 @@ This Python script uses a [🤗 Transformers pipeline](https://huggingface.co/do
 
 ## Embed Streamlit Spaces on other webpages
 
-You can use the HTML `<iframe>` tag to embed a Streamlit Space as an inline frame on other webpages. Simply include the URL of your Space, ending with the `.hf.space` suffix. To find the URL of your Space, you can use the "Embed this Space" button from the Spaces options.
-
-For example, the demo above can be embedded in these docs with the following tag:
+You can use the HTML `<iframe>` tag to embed a Streamlit Space as an inline frame on other webpages. Simply include the URL of your Space, ending with the `.hf.space` suffix. To find the URL of your Space, you can use the "Embed this Space" button from the Spaces options. Then you get the following code snippet which you directly can use in your homepage:
 
 ```
 <iframe
-  src="https://NimaBoscarino-hotdog-streamlit.hf.space?embed=true"
-  title="My awesome Streamlit Space"
+	src="https://kornia-kornia-augmentations-tester.hf.space"
+	frameborder="0"
+	width="850"
+	height="450"
 ></iframe>
 ```
 
-<!-- The height of this iframe has been calculated as 236 + 64 * 2. 236 is the inner content height measured with Chrome 108. 64 is padding-top of its container element. -->
 <iframe
-  src="https://NimaBoscarino-hotdog-streamlit.hf.space?embed=true"
-  frameborder="0"
-  height="364"
-  title="Streamlit app"
-  class="container p-0 flex-grow space-iframe"
+	src="https://kornia-kornia-augmentations-tester.hf.space"
+	frameborder="0"
+	width="850"
+	height="450"
+    class="container p-0 flex-grow space-iframe"
   allow="accelerometer; ambient-light-sensor; autoplay; battery; camera; document-domain; encrypted-media; fullscreen; geolocation; gyroscope; layout-animations; legacy-image-formats; magnetometer; microphone; midi; oversized-images; payment; picture-in-picture; publickey-credentials-get; sync-xhr; usb; vr ; wake-lock; xr-spatial-tracking"
   sandbox="allow-forms allow-modals allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts allow-downloads"
 ></iframe>
 
-Please note that we have added `?embed=true` to the URL, which activates the embed mode of the Streamlit app, removing some spacers and the footer for slim embeds.
-
```

Comment:
is this not relevant anymore? (im not sure)

Reply:
Not when when i updated the hf spaces in kornias docs as e.g. here (it is live): https://github.com/kornia/kornia/pull/2964/files#diff-bec16dbe99a3892599bb805746d7f3e56135fe05df0166f4fc2d86cf9ab22b5e

Also checked here: https://huggingface.co/docs/hub/spaces-embed#embedding-with-iframes where it also does not seem to be necessary.